app/controllers/inventories_controller.rb

```diff
@@ -0,0 +1,5 @@
+class InventoriesController < ApplicationController
+  def index; end
+
+  def show; end
+end
```

Comment:
- [ ] Kindly add the `new`, `create`, and `destroy` actions as well for they are required in the project requirement.
- [ ] Kindly add to logic in the action methods. For example,
```
def index
   @inventory = Inventory.where(...)
end
```
This will help you pass the data to the views.